crates/chain/src/rusqlite_impl.rs

```diff
@@ -533,10 +535,28 @@ impl keychain_txout::ChangeSet {
         )
     }
 
+    /// Get v1 of sqlite [keychain_txout::ChangeSet] schema
+    pub fn schema_v1() -> String {
+        format!(
+            "CREATE TABLE {} ( \
+            descriptor_id TEXT NOT NULL REFERENCES {}, \
```

Comment:
While testing I ran into a failed foreign key constraint because bdk attempted to write to the derived spks table before any scripts were revealed hence the last_revealed table was empty.
```suggestion
            descriptor_id TEXT NOT NULL, \
```